src/main/scopes/wc/wc-scope.ts

```diff
@@ -226,10 +228,12 @@ export class WcScope extends Scope {
 
         for (const importPath of indexImports) {
           const segments = importPath.split('/')
-          const componentName = segments[segments.length - 1]?.replace(/\.js$/, '') ?? ''
+          const fileName = segments[segments.length - 1]?.replace(/\.js$/, '') ?? ''
+          const prefix = getWcPrefix(segments)
+          const componentName = `${prefix}-${fileName}`
```

Comment:
```suggestion
          const componentName = prefix ? `${prefix}-${fileName}` : fileName
```
Adding this check just to make sure we don't get a phantom prefix component (i.e. `-button`)